cmd/testclient/client.go

```diff
@@ -276,6 +279,36 @@ func main() {
 			os.Exit(1)
 		}
 		sugar.Infow("DeleteCluster method", "response", v)
+	case "DeleteAllClustersInRegion":
+		v, err := client.GetClusters(context.Background(), getClustersReq)
+		if err != nil {
+			sugar.Errorw("error getting clusters", "account", getClustersReq.AccountName, "provider", getClustersReq.Provider, "region", getClustersReq.Region, "error", err)
+			os.Exit(1)
+		}
+		clusters := make([]string, 0)
+		for _, cluster := range v.Clusters {
+			clusters = append(clusters, cluster.Name)
+		}
+		logger.Sugar().Infow("deleting following clusters", "clusters", clusters)
+
+		for _, cluster := range clusters {
```

Comment:
can we use v.Clusters directly here ? the above looping isnt required i think

Reply:
The above looping is just to log all the clusters we are deleting for user to see and terminate if something doesn't look good. Ideally should be taking a stdin input to confirm but we are not for now.

Reply:
we can log `v.Clusters` right ?

Reply:
v.Clusters is a huge json and difficult to read.